pkgs/development/libraries/qcustomplot/default.nix

```
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitLab,
+  fetchurl,
+  qtbase,
+  qmake,
+}:
+
+stdenv.mkDerivation (finalAttr: {
+  pname = "qcustomplot";
+  version = "2.1.1";
+
+  srcs = [
+    (fetchFromGitLab {
+      owner = "ecme2";
+      repo = "QCustomPlot";
+      tag = "v${finalAttr.version}";
+      hash = "sha256-BW8H/vDbhK3b8t8oB92icEBemzcdRdrIz2aKqlUi6UU=";
+    })
+    (fetchurl {
+      url = "https://www.qcustomplot.com/release/${finalAttr.version}/QCustomPlot-source.tar.gz";
+      hash = "sha256-Xi0i3sd5248B81fL2yXlT7z5ca2u516ujXrSRESHGC8=";
+    })
+  ];
+
+  sourceRoot = ".";
+
+  buildInputs = [ qtbase ];
+
+  nativeBuildInputs = [
+    qmake
+  ];
```

```suggestion
  nativeBuildInputs = [
    qmake
  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
    fixDarwinDylibNames
  ];
```

Doesn't work without fixing `libqcustomplot.2.1.1.dylib` on darwin.

---

Sorry for that I don't have a darwin machine to test it, could you please help me.

---

Applying the above should fix the issue. I can also push these changes onto your branch if you prefer (although I'm apparently not allowed to at the moment).

---

> Applying the above should fix the issue. I can also push these changes onto your branch if you prefer (although I'm apparently not allowed to at the moment).

I'm testing these changes by GitHub CI in my own NUR repo, I'll push it later if it all passes.

Thanks for your help ^_^